src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php

```diff
@@ -16,9 +16,9 @@ trait GuardsAttributes
     /**
      * The attributes that aren't mass assignable.
      *
-     * @var array
+     * @var array|bool
      */
-    protected $guarded = ['*'];
+    protected $guarded = false;
```

Comment:
Maybe it's better to make `$guarded = [];` to not mix types?

Reply:
Mixing types seems a bit confusing. Currently, the model is `totallyGuarded()` when `$this->getGuarded() == ['*']` (and without fillable attributes). When `$guarded` is a boolean, and `false` by default, it would give the impression that the model is `totallyGuarded()` when `$guarded` is set to `true`. This could be resolved by documentation, but I agree it could be better not to mix types.